Tests/DashboardPageE2ETests.cs

```diff
@@ -178,193 +152,149 @@ public void DashboardPageFilterDropdownShouldSelectNoBudgetHotelsIfNoBudgetOptio
             hotelFilters.Single(option => option.Text == "Local").Click(); // Deselect local
             Thread.Sleep(1000); // wait to ensure local option are selected then deselected
-            hotelFilters = wait.Until(webDriver => webDriver.FindElements(By.ClassName("HotelOptions")));
+            hotelFilters = _wait.Until(webDriver => webDriver.FindElements(By.ClassName("HotelOptions")));
             IEnumerable<IWebElement> filters =
                 hotelFilters.Where(option => option.GetAttribute("class").Contains("selected")).ToList();
             bool noOptionsSelected = filters.IsNullOrEmpty();
             
             Assert.True(noOptionsSelected);
-            Teardown(driver);
         }
         
         [Fact]
         public void DashboardPageFilterDropdownShouldDeselectNoBudgetHotelsIfNoBudgetOptionIsPressedAfterSelectingNoBudgetOption()
         {
-            using IWebDriver driver = Setup();
-            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
-            
-            wait.Until(webDriver => webDriver.FindElement(By.Id("selected-hotels-button"))).Click();
+            _wait.Until(webDriver => webDriver.FindElement(By.Id("selected-hotels-button"))).Click();
             Thread.Sleep(1000); // wait to ensure all options are displayed
             ReadOnlyCollection<IWebElement> hotelFilters =
-                wait.Until(webDriver => webDriver.FindElements(By.ClassName("HotelOptions")));
+                _wait.Until(webDriver => webDriver.FindElements(By.ClassName("HotelOptions")));
             hotelFilters.Single(option => option.Text == "No budget").Click(); // Select no budget
             hotelFilters.Single(option => option.Text == "No budget").Click(); // Deselect no budget
             Thread.Sleep(1000); // wait to ensure no budget option are selected then deselected
-            hotelFilters = wait.Until(webDriver => webDriver.FindElements(By.ClassName("HotelOptions")));
+            hotelFilters = _wait.Until(webDriver => webDriver.FindElements(By.ClassName("HotelOptions")));
             IEnumerable<IWebElement> filters =
                 hotelFilters.Where(option => option.GetAttribute("class").Contains("selected")).ToList();
             bool noOptionsSelected = filters.IsNullOrEmpty();
             
             Assert.True(noOptionsSelected);
-            Teardown(driver);
         }
 
         [Fact]
         public void DashboardPageCalendarForwardArrowShouldMoveCalendarOneMonthForward()
         {
-            using IWebDriver driver = Setup();
-            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
-            
-            wait.Until(webDriver => webDriver.FindElement(By.Id("next-month-button"))).Click();
+            _wait.Until(webDriver => webDriver.FindElement(By.Id("next-month-button"))).Click();
             Thread.Sleep(1000); // wait to ensure new month is displayed
```

Comment:
no way at vi kan tjekke om det er displayed? i stedet for at vente.

Reply:
maybe

Reply:
Nope.